java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S3752_java.html

```diff
@@ -0,0 +1,26 @@
+<p>A <code>@RequestMapping</code> method handles all matching requests by default. That means that a method you intended only to be
```

Comment:
If you talk with @agigleux and decide that we should decide to recommend usage of `@PostMapping`, don't forget to update the rule description as well as the message

Reply:
Alex's rule description is compatible with older version of Spring. About @PostMapping we are going to enforce it using another rule: RSPEC-4488